CHANGES_tcell.md

```diff
@@ -0,0 +1,56 @@
+# Change from termbox to tcell
+
+Original GOCUI was written on top of [termbox](https://github.com/nsf/termbox-go) package. This document describes changes which were done to be able to use to [tcell/v2](https://github.com/gdamore/tcell) package.
+
+## Attribute color
```

Comment:
```suggestion
## `gocui.Attribute` color
```
@mjarkk should it be like this?

Reply:
Actually thinking this might stay as is in the title. As the other titles incorporates attribute word too. What you think?

Reply:
Ya that's a good idea lets go for that.